app/Domain/Integrations/Models/IntegrationUrlModel.php

```diff
@@ -29,10 +30,17 @@ final class IntegrationUrlModel extends UuidModel
     protected static function booted(): void
     {
         self::created(
-            static fn (IntegrationUrlModel $integrationUrlModel) => IntegrationUrlCreated::dispatch(
-                Uuid::fromString($integrationUrlModel->id),
-                Uuid::fromString($integrationUrlModel->integration_id)
-            )
+            static function (IntegrationUrlModel $integrationUrlModel) {
+                // @phpstan-ignore-next-line
+                $id = $integrationUrlModel->id instanceof LazyUuidFromString ? $integrationUrlModel->id : Uuid::fromString($integrationUrlModel->id);
```

Comment:
What is causing this `id` to have type `LazyUuidFromString`?

Reply:
We investigated this, but it looks like an implementation from the uuid lib detail rather than our code

Reply:
Could it be that the new integration URLs sometimes have an empty ID?